src/index.html

```diff
@@ -1,16 +1,40 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+    method="post"
+    class="container">
+      <span class="loop"></span>
+      <input type="text"
+      name="big_input"
+      data-qa="big"
+      class="big input"
+      placeholder="Try &quot;Los Angeles&quot;">
```

Comment:
You need to check and fix it everywhere
<img width="1062" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/39610f42-6b09-406f-a7c0-42e721cba7be">